### PR TITLE
CDPAM-1353 | Support authenticated http proxy for CCMv2. Modifies the…

### PR DESCRIPTION
**Pull request by SachinHg**
… http proxy url to have username:password required for ccmv2 proxy support. Modified command This was tested on an environment

### CODE REVIEW

File: saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh [hunk 1]
```diff
@@ -191,7 +191,11 @@ setup_ccmv2() {
   INVERTING_PROXY_FULL_URL="https://$INVERTING_PROXY_URL/"
 
   if [[ "$IS_PROXY_ENABLED" == "true" ]]; then
-    HTTP_PROXY_URL="http://$PROXY_HOST:$PROXY_PORT"
+    if [[ ! -z ${PROXY_USER} ]]; then
```
Comment (lacikaaa) [inline saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh:194]: you could use `-n` or just flip the branches so you could drop `!`

Comment (SachinHg) [reply saltstack/base/salt/prerequisites/usr/bin/user-data-helper.sh]: thanks @lacikaaa . Did the change.

**Review by zabhi — APPROVED** [review]

**Review by lacikaaa — CHANGES_REQUESTED** [review]
thanks, please squash your commits

**Comment by SachinHg** [comment]
> thanks, please squash your commits

I will squash the commits before I merge the PR.

**Review by lacikaaa — APPROVED** [review]

**Pull request merged**